src/plugins/usage_collection/server/usage_counters/usage_counter.test.ts

```diff
@@ -5,13 +5,14 @@
  * in compliance with, at your election, the Elastic License 2.0 or the Server
  * Side Public License, v 1.
  */
-import { UsageCounter, CounterMetric } from './usage_counter';
+import { UsageCounter } from './usage_counter';
+import type { UsageCounters } from '../../common/types';
```

Comment:
The type and implementation having the same name _might_ not be appropriate. I'm open to renaming the _type_, if needed.